phpstan.neon

```diff
@@ -1,5 +1,5 @@
 parameters:
-    level: 8
+    level: 2
```

Comment:
what happened here? shouldnt it be 7?

Reply:
I think the conflict resolution was the wrong way, just use what was already in this branch and worked (highest possible).

Reply:
Okey.